build.gradle

```diff
@@ -20,7 +20,7 @@ if (!JavaVersion.current().java8Compatible) {
 }
 
 task wrapper(type: Wrapper) {
-	gradleVersion = "2.11"
+	gradleVersion = "3.3"
```

Comment:
With this I was able to run `./gradlew wrapper` and it updated `gradle-wrapper.properties` to 3.3 for me. I'm not a wrapper expert, but otherwise I think I would have had to do `brew install gradle; gradle wrapper`. But with that said, I just updated what was already here.